packages/@aws-cdk/aws-location-alpha/rosetta/default.ts-fixture

```diff
@@ -3,6 +3,7 @@ import { Stack } from 'aws-cdk-lib';
 import { Construct } from 'constructs';
 import * as location from '@aws-cdk/aws-location-alpha';
 import * as iam from 'aws-cdk-lib/aws-iam';
+import * as kms from 'aws-cdk-lib/aws-kms';
```

Comment:
is this needed due to the kms usage in README?

Reply:
Yes. I've added it for use below. Please let me know if it's unnecessary.

https://github.com/aws/aws-cdk/pull/30711/files#diff-c19c71d118fc0ea751b26e880adf196cf143ae23a884327f584fca63f6cc7318R60